pyuvdata/uvdata/tests/test_mir.py

```diff
@@ -228,7 +217,13 @@ def test_mir_remember_me_record_lengths(mir_data_object):
     mir_data = mir_data_object
 
     # Check to make sure we've got the right number of records everywhere
-    assert len(mir_data.ac_read) == 2
+
+    # ac_read only exists if read_auto=True
+    if hasattr(mir_data, "ac_read"):
```

Comment:
Assuming you implement the comment above, this will need to be replaced with `if mir_data.ac_read is not None:`.